src/nwb_benchmarks/benchmarks/streaming_base.py

```diff
@@ -0,0 +1,101 @@
+"""Base template classes for basic benchmarks for NWB streaming read."""
+import fsspec
+import h5py
+import pynwb
+import remfile
+from fsspec.asyn import reset_lock
+
+
+class FileReadStreamingBase:
+    """Base class for basic benchmarks for streaming an NWB file from the DANDI archive."""
+    def setup(self):
+        # Random IBL raw data file; not that many groups
+        self.s3_url = "https://dandiarchive.s3.amazonaws.com/blobs/8c5/65f/8c565f28-e5fc-43fe-8fb7-318ad2081319"
```

Comment:
```suggestion
class FileReadStreamingBase:
    """Base class for basic benchmarks for streaming an NWB file from the DANDI archive."""
    s3_url: str
    
    def setup(self):
        assert getattr(self, "s3_url") is not None
```

Here's what I would do instead; that way in the tests all you need to do to test on a different asset is

```python
class FileReadStreamingCase1(FileReadStreamingBase):
    """A basic benchmark for streaming an NWB file from the DANDI archive."""
    s3_url = "a particular s3 URL"
    repeat = 1
```

Reply:
Good idea. Done.